tests/test_os_events.py

```
@@ -0,0 +1,282 @@
+    )
+
+    try:
+        await asyncio.wait_for(got_wrong_kind.wait(), timeout=1.0)
+    except asyncio.TimeoutError:
+        pass  # expected: wrong kind must not arrive
+
+    task.cancel()
+    try:
+        await task
+    except (asyncio.CancelledError, Exception):
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)